resolwe_bio/processes/workflows/rnaseq_variant_calling.py

```diff
@@ -0,0 +1,400 @@
+
+    class Output:
+        """Output fields."""
+
+        # Workflow do not have outputs.
+
+    def run(self, inputs, outputs):
+        """Run the workflow."""
+
+        if inputs.preprocessing:
+
+            input_preprocessing = {
+                "reads": inputs.reads,
+                "reference": {
+                    "sequences": inputs.bbduk.adapters or [],
+                    "literal_sequences": inputs.bbduk.custom_adapter_sequences,
+                },
+                "operations": {
+                    "quality_encoding_offset": inputs.bbduk.quality_encoding_offset,
+                    "ignore_bad_quality": inputs.bbduk.ignore_bad_quality,
+                },
+            }
+
+            if inputs.reads.type.startswith("data:reads:fastq:single:"):
+                slug_bbduk = "bbduk-single"
+            elif inputs.reads.type.startswith("data:reads:fastq:paired:"):
+                slug_bbduk = "bbduk-paired"
+            else:
+                self.error("Wrong reads input type.")
+
+            preprocessing = Data.create(
+                process=BioProcess.get_latest(slug=slug_bbduk),
+                input=input_preprocessing,
+            )
+
+        input_star = {
+            "genome": inputs.genome,
+            "two_pass_mapping": {
+                "two_pass_mode": inputs.alignment.two_pass_mode,
+            },
+            "output_options": {
+                "out_unmapped": inputs.alignment.out_unmapped,
+            },
+        }
+
+        if inputs.preprocessing:
+            input_star["reads"] = preprocessing
+        else:
+            input_star["reads"] = inputs.reads
+
+        alignment = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input=input_star,
+        )
+
+        mark_duplicates = Data.create(
+            process=BioProcess.get_latest(slug="markduplicates"),
+            input={"bam": alignment},
+        )
+
+        split_ncigar = Data.create(
+            process=BioProcess.get_latest(slug="gatk-split-ncigar"),
+            input={
+                "bam": mark_duplicates,
+                "ref_seq": inputs.ref_seq,
+            },
+        )
+
+        input_bqsr = {
+            "bam": split_ncigar,
+            "reference": inputs.ref_seq,
+            "known_sites": [inputs.dbsnp],
+            "read_group": inputs.read_group,
+            "advanced": {
+                "use_original_qualities": True,
+            },
+        }
+
+        if inputs.intervals_bed:
+            input_bqsr["intervals"] = inputs.intervals_bed
+
+        if inputs.indels:
+            input_bqsr["known_sites"].append(inputs.indels)
+
+        bqsr = Data.create(
+            process=BioProcess.get_latest(slug="bqsr"),
+            input=input_bqsr,
+        )
+
+        input_hc = {
+            "alignment": bqsr,
+            "genome": inputs.ref_seq,
+            "dbsnp": inputs.dbsnp,
+            "mbq": 10,
```

Comment:
Should this be replaced with stand_call_conf. It seems that best practices use `--standard-min-confidence-threshold-for-calling` flag.

Reply:
I will add `--standard-min-confidence-threshold-for-calling` flag. The option `mbq` is added here, so that it is set to the [default](https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller#--min-base-quality-score) value of the actual tool. The default value in the process `vc-gatk4-hc` is different.